libs/s25main/desktops/dskGameInterface.cpp

```diff
@@ -684,45 +515,251 @@ bool dskGameInterface::Msg_LeftDown(const MouseCoords& mc)
         // aktuelle Mausposition merken, da diese durch das Schließen verändert werden kann
         if(actionwindow)
             actionwindow->Close();
-        VIDEODRIVER.SetMousePos(mc.GetPos());
+        VIDEODRIVER.SetMousePos(mc.pos);
+
+        ShowActionWindow(action_tabs, cSel, mc.pos, enable_military_buildings);
+    }
+
+    return true;
+}
+
+void dskGameInterface::Msg_ButtonClick(const unsigned ctrl_id)
```

Comment:
Did you move those methods around? Any reason for that? The diff is very large and I can't tell if there were changes.